src/lib/interfaces/uring_drive_interface.cpp

```diff
@@ -128,7 +128,7 @@ static void prep_sqe_from_iocb(drive_iocb* iocb, struct io_uring_sqe* sqe) {
 }
 
 bool uring_drive_channel::can_submit() const {
```

Comment:
I believe in this way, during execution the value can change by user and can cause queue overflow. So, this only needs to be used only when `io_uring_queue_init` called and remaines unchanged after it.